### ISSUE TITLE
Emitted events should have a contract relationship

### ISSUE THREAD

**Issue by mttrbrts** [accordproject/ergo#200]
Once this PR is merged into cicero models, https://github.com/accordproject/models/pull/4

The runtime should automatically add the value of the governing contract relationship if it is not specified in the source code.

**Comment by mttrbrts** [comment]
See Obligation model here: https://models.accordproject.org/cicero/runtime.html
